opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
```

Comment:
*please* no OPM-specfic hacks, there are enough of these already. (e.g. @joakim-hove's proposal may lead to naming conflicts if no extra precautions are taken.) better use boost here:

https://stackoverflow.com/questions/1727881/how-to-use-the-pi-constant-in-c

and

http://www.boost.org/doc/libs/1_61_0/libs/math/doc/html/math_toolkit/constants.html